app/scripts/lib/util.js

```diff
@@ -149,12 +138,42 @@ function checkForError () {
   return new Error(lastError.message)
 }
 
+/**
+ * Returns string with '0x' hex prefix
+ *
+ * @param {sring} str
+ * @returns {string|Object}
```

Comment:
When would this return an `Object`? :thinking:

Reply:
When `str` param will be an `Object`, but I guess it could be any type in this case. So I'll just remove `Object` option.